### PR TITLE
Add placeholder for missing videostream

### LINKED ISSUE #460: Add placeholder when videostream is missing
**Describe the improvement you would like to see**
Similar to what is done with `No ongoing missions` placeholder, add a placeholder when the videostream is missing.

### PR DESCRIPTION
**Pull request by MortFred**
Closes #460

### CODE REVIEW

File: frontend/src/components/VideoStream/VideoSection.tsx [hunk 1]
```diff
@@ -23,15 +25,37 @@ const FullscreenButton = styled(Button)`
...
+                )}
+            {!ValidateVideoStream(videoStream) && <NoVideoPlaceholder name="videocam_off" />}
         </StyledVideoSection>
     )
 }
+
+function ValidateVideoStream(videoStream: VideoStream) {
+    const stream = new Image()
+    stream.src = videoStream.url
+    if (stream.naturalHeight === 0) {
```
Comment (GodVenn) [inline frontend/src/components/VideoStream/VideoSection.tsx:57]: Is this a safe check for validating video stream?

Comment (MortFred) [reply frontend/src/components/VideoStream/VideoSection.tsx:57]: Yes, if the image is available then it must return naturalHeight > 0.
However, I don't have a video stream stack set up on my setup, so I won't be able to check for edge cases.
If we find some, we can always augment the validate function was my thinking.

Comment (GodVenn) [reply frontend/src/components/VideoStream/VideoSection.tsx:57]: Okidoki

**Review by GodVenn — APPROVED** [review]
LGTM

**Pull request merged**